src/test_data/users.js

```diff
@@ -0,0 +1,7 @@
+export const users = [
```

Comment:
I would use an object here

```
export const users = {
    standardUser: {
        username: 'standard_user',
        password: 'secret_sauce',
        role: 'standard',
    },
};
```

And use it like this - users.standardUser.username. The array is not convenient here if you have more than one user

Reply:
I did this on purpose, because in a real project there can be more than one user and they can have different roles